### DISCUSSION TITLE
How to run the openai whisper large model in multiple devices GPU

### DISCUSSION THREAD

**Discussion by Arasun01** [openai/whisper#1610] [Q&A]
I am using a whisper-large v2 model in a single GPU (NVIDIA Tesla V100) computing environment. The performance of the model is a bit slow. For example, while running the 27 mins audio file it is taking more than 10 mins. How do I run the parallel processing or multiple GPU environment? Could someone provide some points for that?

I would like to reduce the execution time as much as possible. How I improve the model execution time?

**Comment by phineas-pta** [comment]
#716
